plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/AppEngineLibraries.java

```diff
@@ -16,29 +16,58 @@
 
 package com.google.cloud.tools.eclipse.appengine.libraries;
 
-import java.util.Arrays;
-import java.util.Collections;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.eclipse.core.runtime.IConfigurationElement;
+import org.eclipse.core.runtime.RegistryFactory;
 
 import com.google.cloud.tools.eclipse.appengine.libraries.model.Library;
+import com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryFactory;
+import com.google.cloud.tools.eclipse.appengine.libraries.model.LibraryFactoryException;
+import com.google.common.collect.ImmutableMap;
 
 public class AppEngineLibraries {
```

Comment:
Maybe we should rename this class to indicate it's more general function. With groups we can define other other than App Engine related libraries in the future, e.g. Cloud SQL libraries.